Robot_Engine/Robot_Engine.csproj

```diff
@@ -56,6 +56,8 @@
     </Reference>
     <Reference Include="Interop.RobotOM">
       <HintPath>C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2017\System\Exe\Interop.RobotOM.dll</HintPath>
+      <HintPath>C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2018\System\Exe\Interop.RobotOM.dll</HintPath>
```

Comment:
The toolkit actually does not compile on my machine after this change, as I do not have Robot 2019 installed and when multiple hintpaths are added without any conditions, it will always use the last.

Condition needs to be added to each of the hintPaths: https://stackoverflow.com/a/26738314

Changing to something like this made it compile on my machine:

```
<HintPath Condition="Exists('C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2017\System\Exe\Interop.RobotOM.dll')">C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2017\System\Exe\Interop.RobotOM.dll</HintPath>
<HintPath Condition="Exists('C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2018\System\Exe\Interop.RobotOM.dll')">C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2018\System\Exe\Interop.RobotOM.dll</HintPath>
<HintPath Condition="Exists('C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2019\System\Exe\Interop.RobotOM.dll')">C:\Program Files\Autodesk\Autodesk Robot Structural Analysis Professional 2019\System\Exe\Interop.RobotOM.dll</HintPath>
```

Reply:
Thanks for spotting this and finding the solution on StackOverflow 👍